include/up-cpp/uri/builder/BuildUUri.h

```diff
@@ -52,14 +52,12 @@ namespace uprotocol::uri {
          * @return 
          */
         auto setAutority(uprotocol::v1::UAuthority const &authority) -> BuildUUri & {
```

Comment:
fix typo

Reply:
I saw that one. It touches several files. Would you mind if I make that change in a separate PR? I think the repos all need a find-and-replace for Autority->Authority.